Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -568,6 +578,16 @@ define([
             }
             that._credit = credit;
 
+            if (defined(properties.customTags)) {
```

Comment:
As a general rule, we try to keep property access to a minimum, rather than have `properties.customTags` multiple times, extract a `var customTags = properties.customTags` variable and use it throughout.

Reply:
ok